pkgs/games/hedgewars/default.nix

```diff
@@ -1,24 +1,23 @@
+
 { SDL_image, SDL_ttf, SDL_net, fpc, qt4 , ghc, ffmpeg, freeglut, network, vector
 , stdenv, makeWrapper, fetchurl, cmake, pkgconfig, lua5_1, SDL, SDL_mixer
-, utf8String, bytestringShow, hslogger, random, dataenc, zlib, libpng, mesa
+, utf8String, bytestringShow, hslogger, random, dataenc, zlib, libpng, mesa, haskellPackages, physfs
```

Comment:
Yes, ` haskellPackages` is dead'ish and will be removed in 2-3 weeks. `haskellngPackages` is the way to go.